src/main/java/com/faforever/api/data/domain/Message.java

```diff
@@ -10,8 +20,18 @@
 @Entity
 @Table(name = "messages")
 @Setter
+@Include(rootLevel = true)
+@SharePermission(expression = IsModerator.EXPRESSION)
+@DeletePermission(expression = IsModerator.EXPRESSION)
+@UpdatePermission(expression = IsModerator.EXPRESSION)
+@CreatePermission(expression = IsModerator.EXPRESSION)
+@Audit(action = Action.DELETE, logStatement = "Message with key `{0}` and id `{1}` deleted", logExpressions = {"${message.key}", "${message.id}"})
+@Audit(action = Action.CREATE, logStatement = "Message with key `{0}` and id `{1}` created", logExpressions = {"${message.key}", "${message.id}"})
```

Comment:
`ID` is uppercase ;)
I assume the "who did it" is logged automatically?

Reply:
Not sure, I guess so, as it is never logged, but i will try